templates/shortcode-form-grid.php

```diff
@@ -452,7 +456,10 @@ static function ($term) use ($tag_text_color, $tag_bg_color) {
                         <div class="form-grid-raised__details">
                             <span class="amount form-grid-raised__details_donations">
                                 <?php
-                                echo give_format_amount($form->get_sales(), ['decimal' => false]) ?>
+                                /**
+                                 * @unreleased Replace $form->get_sales with (new Give\MultiFormGoals\ProgressBar\Model(['ids' => [$form->ID]]))->getDonationCount()
+                                 */
+                                echo (new Give\MultiFormGoals\ProgressBar\Model(['ids' => [$form->ID]]))->getDonationCount() ?>
```

Comment:
I'm seeing a few instances of `$form->get_sales()` in this file - should we be concerned about that?  If so, we might want to either modify the the original method or add a filter so we can modify on demand.

Reply:
Great catch! I liked the filter idea. Resolved in a31ea7d3c217831efb091b834bb39e832992c217